_docs_integrate/13-connector-error-codes.md

```diff
@@ -0,0 +1,79 @@
+| {% include custom-anchor name="error.transport.general.platformClientSecretNotSet" %}                                 | The platform client secret was not set.                                                                                                                                   |
+| {% include custom-anchor name="error.transport.identity.realmLength" %}                                               | The given realms is not 3 characters long.                                                                                                                                |
+| {% include custom-anchor name="error.transport.incompatibleBackbone" %}                                               | The backbone sent an invalid payload.                                                                                                                                     |
+| {% include custom-anchor name="error.transport.messages.noMatchingRelationship" %}                                    | No Relationship was found for a specific sender.                                                                                                                          |
+| {% include custom-anchor name="error.transport.messages.ownAddressNotInList" %}                                       | The recipients list of a message didn't contain an entry for the own address. This message should not have been received.                                                 |
+| {% include custom-anchor name="error.transport.messages.plaintextMismatch" %}                                         | The own address was not named as a recipient within the signed MessagePlaintext. For example this can be caused by a replay attack.                                       |
+| {% include custom-anchor name="error.transport.messages.signatureListMismatch" %}                                     | The signature list didn't contain an entry for a given address.                                                                                                           |
+| {% include custom-anchor name="error.transport.messages.signatureNotValid" %}                                         | The digital signature on a message for a peer key is invalid. An impersonination attack might be the cause of this.                                                       |
+| {% include custom-anchor name="error.transport.notSupported" %}                                                       | The requested method is not yet supported.                                                                                                                                |
+| {% include custom-anchor name="error.transport.recordNotFound" %}                                                     | The requested entity was not found.                                                                                                                                       |
```

Comment:
*record*NotFound => The requested *entity* was not found

Reply:
gibt es in transport nichtmal